_search-plugins/knn/knn-index.md

```diff
@@ -109,13 +111,45 @@ For more information about setting these parameters, please refer to [*faiss*'s
 
 The IVF algorithm requires a training step. To create an index that uses IVF, you need to train a model with the
 [Train API]({{site.url}}{{site.baseurl}}/search-plugins/knn/api#train-model), passing the IVF method definition. IVF requires that, at a minimum, there should be `nlist` training
-data points, but it is [recommended to use more](https://github.com/facebookresearch/faiss/wiki/Guidelines-to-choose-an-index#how-big-is-the-dataset).
-Training data can either the same data that is going to be ingested or a separate set of data.
+data points, but it is [recommended that you use more](https://github.com/facebookresearch/faiss/wiki/Guidelines-to-choose-an-index#how-big-is-the-dataset).
+Training data can be composed of either the same data that is going to be ingested or a separate dataset.
+
+### Supported Lucene methods
+
+Method Name | Requires Training? | Supported Spaces | Description
+:--- | :--- | :--- | :---
+`hnsw` | false | l2, innerproduct, cosinesimil | Hierarchical proximity graph approach to Approximate k-NN search.
```

Comment:
Please remove 'innerproduct' from set of supported methods. It's recent change that eng team done on 8/3.

Reply:
@martin-gaievski I've removed `innerproduct` from the table above. I also made appropriate changes in the [Spaces](http://localhost:4000/docs/latest/search-plugins/knn/approximate-knn/#spaces) table in the Approximate k-NN search section. It seemed easiest to just add "Not supported for Lucene" in parentheses below it, since it's still good for nmslib and faiss. In the same Spaces table, should I also do this for `l1 `and `linf`?